applications_tests/dem_3d/packing_in_cylinder.output

```diff
@@ -6,107 +6,109 @@ Starting simulation with Lethe/DEM on 1 processors
 Reading triangulation 
 
 Finished reading triangulation 
+Warning: expansion of particle-wall contact list is enabled. 
+This feature should only be activated in geometries with concanve boundaries. (For example, for particles flow inside a cylinder or sphere). In geometries with convex boundaries, this feature MUST NOT be activated.
```

Comment:
I would do it the other way around. Enable it by default and if someone disables it , it will be because they know what they are doing and Then you can print this warning.

Reply:
I changed the default value to true. But I think it's better to keep this warning with the current format because users could disable the feature if they have concave boundaries.